pegen/pegen.c

```diff
@@ -772,3 +772,53 @@ map_targets_to_del_names(Parser *p, asdl_seq *seq)
     }
     return new_seq;
 }
+
+KeyValuePair *
+key_value_pair(Parser *p, expr_ty key, expr_ty value)
```

Comment:
Nit: For consistency with other functions in this file, we should add a comment before the function as documentation. Example of what I mean:

```
/* Creates an asdl_seq* where all the elements have been changed to have del as context */
asdl_seq *
map_targets_to_del_names(Parser *p, asdl_seq *seq)	map_targets_to_del_names(Parser *p, asdl_seq *seq)
{
 int len = asdl_seq_LEN(seq);
```